pkg/cloudprovider/providers/vsphere/vsphere_util.go

```diff
@@ -526,3 +491,30 @@ func (vs *VSphere) checkDiskAttached(ctx context.Context, nodes []k8stypes.NodeN
 	}
 	return nodesToRetry, nil
 }
+
+func (vs *VSphere) IsDummyVMPresent(vmName string) (bool, error) {
```

Comment:
Let's add an empty line in between separate semantic blocks, e.g. line 499/500, 503/504, 507/508, etc. to improve code readability a little bit.

Reply:
Done